packages/@webex/plugin-cc/test/unit/spec/services/core/WebSocket/connection-service.ts

```diff
@@ -0,0 +1,50 @@
+import { ConnectionService } from '../../../../../../src/services/core/WebSocket/connection-service';
+import { WebSocketManager } from '../../../../../../src/services/core/WebSocket/WebSocketManager';
+import { Signal } from '../../../../../../src/services/core/Signal';
+
+jest.mock('../../../../../../src/services/core/WebSocket/WebSocketManager');
+
+describe('ConnectionService', () => {
```

Comment:
Mostly taken tests from the agent desktop in this case. All test cases are passing.